DataFormats/EgammaCandidates/interface/Photon.h

```diff
@@ -471,8 +471,53 @@ namespace reco {
     float pfMVA() const {return pfID_.mva;}
     // setters
     void setPflowIDVariables ( const PflowIDVariables& pfid ) {  pfID_ = pfid;}     
+    
+    struct PreshowerShapes { 
+      float effSigmaRR;
```

Comment:
is it worth it: a struct with one data member?
Can this just go to ShowerShapes as well?